FWCore/Framework/src/EventProcessor.cc

```diff
@@ -486,13 +483,13 @@ namespace edm {
     ScheduleItems items;
 
     //initialize the services
-    std::shared_ptr<std::vector<ParameterSet> > pServiceSets = processDesc->getServicesPSets();
-    ServiceToken token = items.initServices(*pServiceSets, *parameterSet, iToken, iLegacy, true);
+    auto pServiceSets = processDesc->getServicesPSets();
```

Comment:
I believe the `p` in `pServiceSets` stood for pointer. Now it could just be `serviceSets`.